src/UI/styles/autofill-tooltip-styles.js

```diff
@@ -116,6 +116,9 @@ module.exports = `
     margin: auto 8px auto 0;
     background-size: cover;
 }
+.tooltip__button--data:hover::before {
+    filter: invert(100%);
+}
```

Comment:
Instead of maintaining the icons in black and in white, I can just invert the colour. 🙂